src/buffer.c

```diff
@@ -5751,8 +5792,8 @@ bt_nofile(buf_T *buf)
 #endif
 
 /*
- * Return TRUE if "buf" is a "nowrite", "nofile", "terminal" or "prompt"
- * buffer.
+ * Return TRUE if "buf" is a "nowrite", "nofile", "terminal", "prompt", or
+ * "popup" buffer.
```

Comment:
Note: This function still works, but I think the fact that it worked for "popup" was completely coincidental, so just updated comment to make it intentional.